lib/src/client.dart

```diff
@@ -58,26 +58,20 @@ class SubiquityClient {
     _client.close();
   }
 
-  Future<T> _receive<T>(
+  Future<R> _receive<R, V>(
```

Comment:
we need two template types because
- the Returned decoded type e.g. TimeZoneInfo, String, bool, ...
- the Value type received over the wire e.g. Map<String, dynamic>, String, bool, ...

can be different. luckily, the Dart compiler is pretty good at inferring generic types so we don't need to explicitly declare the types anywhere.